ui/app/mainui/AppMain.qml

```diff
@@ -818,14 +818,10 @@ Item {
         }
 
         function openShell() {
-            shellLoader.active = true
+            appMain.rootStore.mainModuleInst.setActiveSectionById("shell")
```

Comment:
```suggestion
            appMain.rootStore.mainModuleInst.setActiveSectionBySectionType(Constants.appSection.shell)
```

I guess this is safer since we don't really know the string ID (coming from NIM)